packages/meditrak-server/src/routes/exportSurveyResponses.js

```diff
@@ -83,6 +85,11 @@ export async function exportSurveyResponses(req, res) {
   try {
     let country;
     let entities;
+    let reportName;
+    if (viewId) {
+      const dashboardReport = await models.dashboardReport.findById(viewId);
+      reportName = dashboardReport.viewJson.name;
+    }
```

Comment:
```suggestion
    const reportName = viewId && (await models.dashboardReport.findById(viewId)).viewJson.name;
```
?

Reply:
Look nicer! That inspires me to refactor the code.

Reply:
Seems like the name couldn't be fetched from the object in this way unfortunately. But I did some refactoring for the code below. Hope that can add value.